### ISSUE TITLE
Add support for `accessibilitySetSize` [Windows Only]

### ISSUE THREAD

**Issue by YajurG** [microsoft/react-native-windows#12512]
Add support for the `accessibilitySetSize` prop.

## Documentation
[accessibilitySetSize documentation](https://microsoft.github.io/react-native-windows/docs/iviewwindowsprops-api)

## Behavior
The `accessibilitySetSize` prop indicates to accessibility services that the Component is within a set with the given size. 

The value of this prop should be used to set the UIA `SizeOfSet` attached property (See [here](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.automationproperties.sizeofset?view=winrt-26100#windows-ui-xaml-automation-automationproperties-sizeofset) for documentation of this property).

## Implementation Plan
We should:
1. Take the value of the `accessibilitySetSize` prop and set the UIA `SizeOfSet` property for that component to be submitted value.

## To Be Clarified
Note this prop should be specified in tandem with the `accessibilityPosInSet` prop. We should consider filing a Yellow or Red Box error if the source only specifies `accessibilityPosInSet`.

**Comment by danielayala94** [comment]
@Yajur-Grover do we still need this issue open? I see a bunch of filed issues to implement accessibilitySetSize for each of the core components. Example: #13164
